src/main/scala/dx/core/ir/TypeSerde.scala

```diff
@@ -89,9 +102,14 @@ object TypeSerde {
     )
   }
 
-  def serializeOne(t: Type): JsObject = {
-    val (typeJs, schemas) = serializeType(t, Map.empty)
-    JsObject("type" -> typeJs, "schemas" -> JsObject(schemas))
+  /**
+    * Serialize and create a specification for a single type.
+    * @param t the type
```

Comment:
This docstring may need to be regenerated because it misses `schemas`

Reply:
fixed